opsdroid/connector/matrix/connector.py

```diff
@@ -216,9 +218,14 @@ async def connect(self):
             self.connection.user_id = self.mxid
 
         elif self.mxid is not None and self.password is not None:
-            login_response = await self.connection.login(
-                password=self.password, device_name=self.device_name
-            )
+            try:
+                login_response = await self.connection.login(
+                    password=self.password, device_name=self.device_name
+                )
+            except:
```

Comment:
Could we do something like this instead?

```suggestion
            except Exception:
```